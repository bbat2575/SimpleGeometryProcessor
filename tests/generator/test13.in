-N=20 -mindist= -rseed=1