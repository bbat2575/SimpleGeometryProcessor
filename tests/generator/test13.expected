usage: gen_points.py [-h] [-N N] [-mindist MINDIST] [-rseed RSEED]
gen_points.py: error: argument -mindist: invalid float value: ''
